src/main/java/build/buildfarm/instance/shard/ShardInstance.java

```diff
@@ -2019,7 +2019,7 @@ private ListenableFuture<Boolean> checkCacheFuture(
     }
 
     Context.CancellableContext withDeadline =
-        Context.current().withDeadlineAfter(60, SECONDS, contextDeadlineScheduler);
+        Context.current().withDeadlineAfter(99, SECONDS, contextDeadlineScheduler);
```

Comment:
```suggestion
        Context.current().withDeadlineAfter(60, SECONDS, contextDeadlineScheduler);
```

Reply:
It looks like this got pushed in a frenzy - sorry for the confusion 🏃